WordPress/Classes/ViewRelated/Domains/Domain registration/RegisterDomainSuggestions/RegisterDomainSuggestionsViewController.swift

```diff
@@ -81,6 +99,43 @@ class RegisterDomainSuggestionsViewController: UIViewController {
         return nil
     }
 
+    // MARK: - View Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configure()
+        hideButton()
+        setupTransferFooterView()
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        if let domainsTableViewController {
+            let insets = UIEdgeInsets(
+                top: 0,
+                left: 0,
+                bottom: transferFooterView.isHidden ? 0 : transferFooterView.bounds.height,
+                right: 0
+            )
+            if insets != domainsTableViewController.additionalSafeAreaInsets {
+                domainsTableViewController.additionalSafeAreaInsets = insets
+            }
+        }
+    }
+
+    // MARK: - Setup subviews
+
+    private func setupTransferFooterView() {
+#if JETPACK
+        guard domainSelectionType == .purchaseFromDomainManagement else {
+            return
+        }
+        self.view.addSubview(transferFooterView)
+        self.transferFooterView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate(transferFooterViewConstraints.visible)
+#endif
```

Comment:
Why was run argument needed instead of checking it through `AppConfiguration`? It doesn't look like a specific class that only targets JP was used here if I am not mistaken.

But even if we needed that, wouldn't it be better to move the check above the function and also at the callsite? That way we wouldn't be calling an empty function on WP target.

Reply:
That method was refactored multiple times, and there was a Jetpack-only code being used there before. But good catch, I'll remove the that macro code.